examples/asr/conf/speech_multitask/fast-conformer_aed.yaml

```diff
@@ -75,15 +83,15 @@ model:
     type: agg  # Can be either bpe (SentencePiece tokenizer) or wpe (WordPiece tokenizer) or `agg` for aggregate tokenizers
     langs:
       spl_tokens: # special tokens model
-        dir: ???
+        dir: ${spl_tokens.dir}
```

Comment:
This subconfig is not passed to model constructor - so this resolution will fail. You need to use code to manually patch the config before calling constructor.

Reply:
mb, thought this is caught by the OmegaConf.